### PR TITLE
Updated TemperatureControl and TemperatureSensor classes

### PR DESCRIPTION
**Pull request by TaylorSmith28**

### CODE REVIEW

File: titration/utils/devices/temperature_control.py [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
+# pylint: disable = too-many-instance-attributes
+
 import time
 
-import adafruit_max31865
-import busio
-import digitalio
 import pandas as pd
-from gpiozero import LED
+from titration.utils import constants
```
Comment (prestoncarman) [inline titration/utils/devices/temperature_control.py:10]: Use device library import.

File: titration/utils/devices/temperature_probe.py [hunk 2]
```diff
@@ -1,15 +1,33 @@
...
-import digitalio
+"""
+The class for the TemperatureProbe
+"""
+
+# pylint: disable = invalid-name, too-many-arguments
 
 from titration.utils import constants
 
+if constants.IS_TEST:
```
Comment (prestoncarman) [inline titration/utils/devices/temperature_probe.py:9]: Use device library import

File: test/utils/devices/temperature_probe_test.py [hunk 3]
```diff
@@ -58,23 +58,5 @@ def test_temperature_probe_resistance_null():
...
-def test_temperature_probe_set_temperature():
-    """
-    The function to test setting the temperature for the mock sensor
-    """
-    temperature_sensor = create_temperature_probe()
-    temperature_sensor.mock_set_temperature(25)
-    assert temperature_sensor.get_temperature()
-
-
-def test_temperature_probe_set_resistance():
```
Comment (prestoncarman) [inline test/utils/devices/temperature_probe_test.py:74]: How can you achieve this test in the new code base?

File: test/utils/devices/temperature_control_test.py [hunk 4]
```diff
@@ -73,15 +55,6 @@ def test_temperature_control_last_temperature():
     temperature_controller.get_last_temperature()
 
 
-def test_temperature_control_activate():
```
Comment (prestoncarman) [inline test/utils/devices/temperature_control_test.py:76]: How can you test this in the new code base?

File: titration/utils/devices/temperature_control.py (hunk 1 above)
Comment (TaylorSmith28) [reply titration/utils/devices/temperature_control.py]: This has been fixed.

File: titration/utils/devices/temperature_probe.py (hunk 2 above)
Comment (TaylorSmith28) [reply titration/utils/devices/temperature_probe.py]: This has been fixed.

File: test/utils/devices/temperature_probe_test.py (hunk 3 above)
Comment (TaylorSmith28) [reply test/utils/devices/temperature_probe_test.py]: This tests the temperature probe mock set resistance function. The function has been removed as you should not be setting a resistance for the probe. The probe returns that information to you. For testing purposes of the device, we can just change the class's variable directly.

File: test/utils/devices/temperature_control_test.py (hunk 4 above)
Comment (TaylorSmith28) [reply test/utils/devices/temperature_control_test.py]: This test has been implemented

**Review by prestoncarman — APPROVED** [review]
LGTM

**Pull request merged**